manifests/prometheus-serviceMonitorKubelet.yaml

```diff
@@ -81,10 +81,10 @@ spec:
     scheme: https
     tlsConfig:
       insecureSkipVerify: true
-  jobLabel: k8s-app
+  jobLabel: app.kubernetes.io/name
   namespaceSelector:
     matchNames:
     - kube-system
   selector:
     matchLabels:
-      k8s-app: kubelet
+      app.kubernetes.io/name: kubelet
```

Comment:
I don't think this works, as the prometheus-operator still creates the kubelet endpoints object only with the `k8s-app: kubelet` object.